wiki/Help_Centre/en.md

```diff
@@ -360,6 +433,14 @@ Check your software of choice to make sure that `osu!.exe` in the directory you
 
 Try closing any extra programs before playing osu! if you find that your scores aren't submitting and you are not having issues with connectivity otherwise.
 
+##### Are the your beatmap statuses correct?
+
+**Sometimes the statuses of your beatmaps can get messed up, meaning the scores you set on them can't submit.**
+
+In the song select menu, check in the top left of the screen for any small icons, for example a ranked beatmap should display a blue arrow icon in the top left. If your beatmaps do not display any icon, or an incorrect icon, then your scores cannot submit. 
```

Comment:
Images would be helpful here

Reply:
Sorry, how do I add an image?

Reply:
https://www.markdownguide.org/basic-syntax/#images-1

also may want to refer to the [ASC criteria on images](https://osu.ppy.sh/wiki/en/Article_styling_criteria/Formatting#images) if you're not already familiar with them

Reply:
Thanks for the help, have added an image here